_typos.toml

```diff
@@ -1,12 +1,17 @@
 [files]
 extend-exclude = [
-    "**/resources/**/*",
```

Comment:
These directories do not exist.

Reply:
Ahh this was copy-pasted from Ruff a long time ago.